src/Microsoft.PowerShell.Security/security/CertificateProvider.cs

```diff
@@ -31,20 +31,18 @@ namespace Microsoft.PowerShell.Commands
 {
```

Comment:
@iSazonov, your last commit had 8 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

CmsMessage cmdlets and Get-PfxCertificate basic tests.Verify Get-PfxCertificate -FilePath
```powershell
Cannot bind argument to parameter 'FilePath' because it is null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\CmsMessage.Tests.ps1: line 22
22:         $cert = Get-PfxCertificate -FilePath $certLocation

```
CmsMessage cmdlets and Get-PfxCertificate basic tests.Verify Get-PfxCertificate -LiteralPath
```powershell
Cannot bind argument to parameter 'LiteralPath' because it is null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\CmsMessage.Tests.ps1: line 27
27:         $cert = Get-PfxCertificate -LiteralPath $certLocation

```
CmsMessage cmdlets and Get-PfxCertificate basic tests.Verify Get-PfxCertificate positional argument
```powershell
Cannot bind argument to parameter 'FilePath' because it is null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\CmsMessage.Tests.ps1: line 32
32:         $cert = Get-PfxCertificate $certLocation

```
CmsMessage cmdlets and Get-PfxCertificate basic tests.Verify Get-PfxCertificate right password
```powershell
Cannot bind argument to parameter 'FilePath' because it is null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\CmsMessage.Tests.ps1: line 38
38:         $cert = Get-PfxCertificate $protectedCertLocation -Password $password

```
CmsMessage cmdlets and Get-PfxCertificate basic tests.Verify Get-PfxCertificate wrong password
```powershell
Expected an exception, with FullyQualifiedErrorId 'GetPfxCertificateUnknownCryptoError,Microsoft.PowerShell.Commands.GetPfxCertificateCommand' to be thrown, but the FullyQualifiedErrorId was 'ParameterArgumentValidationErrorNullNotAllowed,Microsoft.PowerShell.Commands.GetPfxCertificateCommand'. from D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\CmsMessage.Tests.ps1:45 char:30
    +         { Get-PfxCertificate $protectedCertLocation -Password $pass - ...
    +                              ~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\CmsMessage.Tests.ps1: line 46
46:             Should -Throw -ErrorId "GetPfxCertificateUnknownCryptoError,Microsoft.PowerShell.Commands.GetPfxCertificateCommand"
```